src/hexkit/providers/akafka/provider/eventsub.py

```diff
@@ -610,39 +639,49 @@ def _validate_extracted_info(self, event: ExtractedEventInfo):
 
     async def _consume_event(self, event: ConsumerEvent) -> None:
         """Consume an event by passing it down to the translator via the protocol."""
-        event_id = get_event_id(event, service_name=self._service_name)
-        event_info = self._extract_info(event)
-        try:
-            self._validate_extracted_info(event_info)
-        except RuntimeError as err:
-            logging.info(
-                "Ignored event of type '%s': %s, errors: %s",
-                event_info.type_,
-                event_id,
-                str(err),
-            )
-            # Always acknowledge event receipt for ignored events
-            await self._consumer.commit()
-            return
+        otel_extractor = AIOKafkaOtelContextExtractor()
+        extracted_context = extract(
+            event.headers,
+            getter=otel_extractor,
+        )
+        with tracer.start_as_current_span(
+            name="Consume event", context=extracted_context
```

Comment:
Maybe it's better to not use spaces in span names, i.e. "consume_event" instead?

https://oneuptime.com/blog/post/2024-11-04-how-to-name-spans-in-opentelemetry/view

Reply:
Thanks for the link 
It seems I also need to change my span naming strategy in the file services